tests/monitoring/prometheus_utils.go

```diff
@@ -205,19 +205,30 @@ func doHttpRequest(url string, endpoint string, token string) *http.Response {
 func getAuthorizationToken(cli kubecli.KubevirtClient, monitoringNs string) string {
 	var token string
 	Eventually(func() bool {
-		var secretName string
 		sa, err := cli.CoreV1().ServiceAccounts(monitoringNs).Get(context.TODO(), "prometheus-k8s", metav1.GetOptions{})
 		if err != nil {
 			return false
 		}
-		for _, secret := range sa.Secrets {
-			if strings.HasPrefix(secret.Name, "prometheus-k8s-token") {
-				secretName = secret.Name
-			}
-		}
+		secretName := fmt.Sprintf("%s-%s-token", sa.Name, monitoringNs)
```

Comment:
Isn't the previous API call redundant?

Reply:
Yes I think I can remove and just use the string.

Reply:
Is this token are going to be deleted upon test suite teardown?